src/guides/v2.3/install-gde/trouble/tshoot_access-browser.md

```diff
@@ -27,9 +27,9 @@ Try the following:
 
 *  Make sure [Apache server rewrites]({{ page.baseurl }}/install-gde/prereq/apache.html) are enabled.
 
-   If Apache server rewrites are set incorrectly, static files are not served from the correct location.
+   If Apache server rewrites are set incorrectly, static files aren't served from the correct location.
```

Comment:
I thought we get rid of contractions?

Reply:
I do not think this is a hard and fast rule. I also did not copyedit every single page in the install guide. I mostly adding content, removed content, moved content around, and restructured topics and TOC. I will make as many of these changes that I can, however.